cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -80,6 +89,7 @@ renderShelleyQueryCmdError err =
     ShelleyQueryCmdHelpersError helpersErr -> renderHelpersError helpersErr
     ShelleyQueryCmdAcquireFailure aqFail -> Text.pack $ show aqFail
     ShelleyQueryCmdByronEra -> "This query cannot be used for the Byron era"
+    ShelleyQueryCmdPoolIdError -> "The pool id does not exist"
```

Comment:
What would be very useful to the user would be to include the `PoolId` and render it here.

Reply:
> @kevinhammond We have a problem with these commands. They both depend on the ledger state query which currently hangs when used on mainnet. We don't have immediate plans to fix this now as the focus is eUTxO things. Context: https://input-output-rnd.slack.com/archives/CC0CLNEH1/p1617065989042000
> 
> > Please avoid in future any production use of the debug state dump features.It's there for debug and hacking / prototyping purposes. But it's not a stable format. It's too big. It's not designed for production use.
> 
> The way forward would likely be bugging the consensus folk to expose a suitable query

Here is the evidence that they do not (run live on mainnet).  The commands deliberately select a subset of the ledger state information, so reducing time and memory requirements for SPOs (who are currently using the ledger-state dump for production reasons).  The commands can, of course, be changed if the ledger state format changes or the consensus team exposes the required state.  The SPOs are very keen to have these available - they will help reduce memory requirements when provisioning a stake pool.

````
time cardano-cli query stake-snapshot --stake-pool-id 00beef0a9be2f6d897ed24a613cf547bb20cd282a04edfc53d477114 --mainnet
{
    "poolStakeGo": 40278547538358,
    "activeStakeGo": 22753958467474959,
    "poolStakeMark": 40424218559492,
    "activeStakeMark": 22670949084364797,
    "poolStakeSet": 39898761956772,
    "activeStakeSet": 22488877070796904
}

real	0m20.085s
user	0m8.874s
sys	0m2.585s
````

````
time cardano-cli query pool-params --stake-pool-id  00beef0a9be2f6d897ed24a613cf547bb20cd282a04edfc53d477114 --mainnet
{
    "poolStakeGo": 40278547538358,
    "activeStakeGo": 22753958467474959,
    "poolStakeMark": 40424218559492,
    "activeStakeMark": 22670949084364797,
    "poolStakeSet": 39898761956772,
    "activeStakeSet": 22488877070796904
}

real	0m20.085s
user	0m8.874s
sys	0m2.585s
````